### ISSUE TITLE
[2.0.1] Top men unbar bar missing after update from 1.2.6

### ISSUE THREAD

**Issue by tuxmaster5000** [thaider/Tweeki#230]
After the update the top bar is missing, Roll-back to 1.2.6 will show it again.
2.0.1
![after](https://user-images.githubusercontent.com/837503/162673644-b42c3c8c-b1d7-4516-bd75-3058dc272b2d.png)
1.2.6
![before](https://user-images.githubusercontent.com/837503/162673646-189d2740-b2f0-42f4-82bf-d3afdbb039bd.png)
Not errors will logged on the browser console.

**Comment by thaider** [comment]
Thanks for reporting. Could you please provide more information about the version of MediaWiki you're using?

**Comment by tuxmaster5000** [comment]
Hi, I use  mediawiki  1.35.6 on PHP  7.4

**Comment by rebastion** [comment]
@tuxmaster5000 I would definitely recommend upgrading MW just to be on the safer side, or wait until November for the next LTS release. I just went thru a few upgrades and test installs with Tweeki and the smoothest option was to work with the latest MW

**Comment by tuxmaster5000** [comment]
We can only use the LTS versions of MW. So I think it will be an nice way to split the Tweeki in to 2 versions. One for the TLS tree of MW and on for the no LTS tree to get an stable LTS support.

**Comment by thaider** [comment]
Sorry for the late response. Are you still having the problem? If so, is your wiki publicly accessible so I can investigate the problem?

**Comment by tuxmaster5000** [comment]
Yes the problem still exists in the next LTS version 1.39.4. The site is public, but I can't activate the theme at any time, because it will breaks the site completely. So I can activate it only in the maintenance window.

**Comment by thaider** [comment]
Could you post the link to your wiki? And do you use any of Tweeki's configuration options in `LocalSettings.php`?

**Comment by tuxmaster5000** [comment]
Today I tested it with 2.0.2, but the same result. Here are the Tweeki settings:

> $wgTweekiSkinHideable = array('firstHeading', 'sidebar-right' );
> $wgTweekiSkinUseBootstrapTheme = true;
> $wgTweekiSkinUseTooltips = true;
> $wgTweekiSkinHideNonAdvanced['EDIT-EXT-special'] = false;
> $wgTweekiSkinHideAnon = array( 'navbar' )
> wfLoadSkin('Tweeki');
> $wgDefaultSkin = 'tweeki';

The URL will be https://pouletpedia.mdc-berlin.de
But  I can only activate the skin in the maintenance window.

**Comment by thaider** [comment]
This is really strange. What's the content of your `MediaWiki:Tweeki-navbar-xxx` system messages? And you don't use `{{#tweekihide:}}` anywhere, right?

**Comment by tuxmaster5000** [comment]
Sorry for the question, but where can I found this? I only manage the server not the content.

**Comment by thaider** [comment]
It's strange your LocalSettings.php actually works because you're missing a semicolon after the line

` $wgTweekiSkinHideAnon = array( 'navbar' )`

What happens if you delete this line?

Also note, that you should rather write `$wgTweekiSkinHideAnon['navbar'] = true;` nowadays, if you wanted to set the configuration option.

**Comment by tuxmaster5000** [comment]
The semicolon was only missing by c&p. Because without it, the whole site will be broken.
But
$wgTweekiSkinHideAnon['navbar'] = true;
will result in the same. :(

**Comment by tuxmaster5000** [comment]
The navbar is also hidden when I'am logged in.
When I switch to the mobile view, that an part of it are show. But the log out and user preferences are complete missing.
